src/desktop/components/article/client/view.coffee

```diff
@@ -208,8 +208,7 @@ module.exports = class ArticleView extends Backbone.View
 
   checkEditable: ->
     if (@user?.get('has_partner_access') and
-       @user?.id is @article.get('author_id')) or
-       @user?.isEditorialAdmin()
+       @user?.id is @article.get('author_id'))
```

Comment:
This is no longer necessary because it only applies to "classic" layout articles, which are no longer used by editorial team.